style.css

```diff
@@ -0,0 +1,166 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700;500&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
+
+.text-decoration-none {
+  text-decoration: none;
+}
+
+.list-style-type-none {
+  list-style-type: none;
+}
+
+.mt-12 {
+  margin-top: 12px;
+}
+
+.weight--400 {
+  font-weight: 400;
+}
+
+.weight--500 {
+  font-weight: 500;
+}
+
+.weight--700 {
+  font-weight: 700;
+}
+
+.justify-content-between {
+  justify-content: space-between;
+}
+
+.justify-content-center {
+  justify-content: center;
+}
+
+.justify-content-around {
+  justify-content: space-around;
+}
+
+.align-items-center {
+  align-items: center;
+}
+
+.align-self-center {
+  align-self: center;
+}
+
+.align-content-center {
+  align-content: center;
+}
+
+.text--primary {
+  color: #6070ff;
+}
+
+.text--info {
+  color: #7f8cff;
+}
+
+.d-none {
+  display: none;
+}
+
+.d-flex {
+  display: flex;
+}
+
+.container {
+  padding: 25px 142px;
+}
+
+.card {
+  height: calc(100vh - 10vh);
+}
+
+.card-title {
+  width: 920px;
+}
+
+/*
+this style here is where you should create your
+custom style should incase is not created
+*/
+.my-logo {
+  font-size: 18px;
+  line-height: 20px;
+}
+
+.nav {
+  background-color: #fff;
+
+  /* box-shadow: 0rem 0rem 1rem #E5E5E5; */
+  height: 72px;
+}
+
+.container .fa-solid {
+  display: none;
+}
+
+.background-image {
+  background-image: url('./assets/images/bg.png');
+  background-repeat: no-repeat;
+  width: auto;
```

Comment:
-  [ OPTIONAL]  Good job adding the proper font as requested by the previous, However in my opinion, you have to use also the proper backgound-image for the mobile section, this is optional for now since it has not been mentioned before. :+1:  
- Your design:
![studentBackground](https://user-images.githubusercontent.com/97834160/205066303-7931d594-e4da-412c-bb29-d50f6a4608ad.png)

- Figma design:
<img src="https://user-images.githubusercontent.com/97100091/205046154-d8a080ce-9f3e-40e3-9544-294670c37c44.png">